docs/docs-ref-autogen/powerpoint/powerpoint/powerpoint.application.yml

```diff
@@ -0,0 +1,81 @@
+### YamlMime:UniversalReference
+items:
+  - uid: 'powerpoint!PowerPoint.Application:class'
+    summary: '\[ [API set: PowerPointApi 1.0](/office/dev/add-ins/reference/requirement-sets/powerpoint-api-requirement-sets) \]'
```

Comment:
According to the PPT team, there is. It's all the core application-specific classes.

Reply:
I'm not sure. I think it helps the tooling. I don't think this is a harmful distinction. We're not going to talk about PPT 1.0 like we do other requirement sets. But since these APIs don't do anything, I think that's fine.

This is probably a larger discussion to have when Ricky returns, but we can always rebrand this API set as needed.